src/SFML/Graphics/Font.cpp

```diff
@@ -155,36 +153,32 @@ bool Font::loadFromFile(const std::filesystem::path& filename)
     // Cleanup the previous resources
     cleanup();
 
-    auto fontHandles = std::make_unique<FontHandles>();
+    auto fontHandles = std::make_shared<FontHandles>();
```

Comment:
This probably still needs to be `shared_ptr` for dependency reasons (only declare destructor in the .cpp file)? Or do we actually use shared semantics?

Reply:
It has always been a `std::shared_ptr`, this PR only makes it more obvious by avoiding a `unique_ptr` to `shared_ptr` conversion. 

I investigated a bit:

- `m_fontHandles` does get copied when `Font` is copied, implying shared semantics.

- Loading the same font multiple times (e.g. using the same path) re-creates a new `FontHandles` shared object. This might be an opportunity for optimization (i.e. caching the `FontHandles` shared pointer).